tools/deb/istio-start.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Script to run Envoy for Istio.
+
```

Comment:
add set -e as for instance if failing to read the sidecar.env it would be dangerous to run the iptables script